domains/certificates/Query.API/API/ContractService/ContractServiceImpl.cs

```diff
@@ -40,11 +40,6 @@ internal class ContractServiceImpl : IContractService
             return new GsrnNotFound();
         }
 
-        if (matchingMeteringPoint.Type != MeterType.Production)
-        {
-            return new NotProductionMeteringPoint();
```

Comment:
Is the type `NotProductionMeteringPoint` deleted?

Reply:
Now it is